pages/auth/forgot-password/[id].tsx

```diff
@@ -0,0 +1,231 @@
+import { getCsrfToken } from "next-auth/client";
+import prisma from "../../../lib/prisma";
+
+import Head from "next/head";
+import React from "react";
+import debounce from "lodash.debounce";
+import dayjs from "dayjs";
+import { ResetPasswordRequest } from "@prisma/client";
+import { useMemo } from "react";
+import Link from "next/link";
+import { GetServerSidePropsContext } from "next";
+
+type Props = {
```

Comment:
Oh I like this; is this the usual way React deals with Prop types?

Reply:
Interfaces work as well but coming from true OO Langs calling using interfaces and types interchangeably doesn't sit well with me. And TS types over prop types only because we're in TS land..